app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
```

Comment:
Thanks for the comment!

I updated the class to `pagination--with-total` and de-nested the `pagination__total` style. LMK if that doesn't look better!